package.json

```diff
@@ -33,12 +33,12 @@
     "consola": "^3.2.3"
   },
   "devDependencies": {
-    "@types/node": "^20.12.7",
```

Comment:
This change is not related to this PR.

Reply:
reverted in this commit: 4c0688ccc28978fb9ea75dd6b97d34fb03be0057